src/generateOutsideCheck.js

```diff
@@ -1,4 +1,21 @@
 /**
+ * Check if classList contains any of the given classes,
+ * transparently handling singular and array cases.
+ */
+function containsAny(classList, classes) {
+  if (Array.isArray(classes)) {
```

Comment:
would be better to just pack in an array if its not one yet and handle only one case below that